### PR TITLE
[rootfs-generator] Add SELinux labeling for update image

### PR DESCRIPTION
**Pull request by LKomaryanskiy**
Signed-off-by: Leonid Komarianskyi <leonid_komarianskyi@epam.com>

### CODE REVIEW

File: classes/rootfs-image-generator.bbclass [hunk 1]
```diff
@@ -82,6 +82,9 @@ do_create_rootfs_image() {
         init_ostree_repo
     fi
 
+    ROOTFS_FULL_PATH=$(realpath ${ROOTFS_SOURCE_DIR})
```
Comment (al1img) [inline classes/rootfs-image-generator.bbclass:85]: It should be done if SE linux enabled. Right?

Comment (al1img) [inline classes/rootfs-image-generator.bbclass:85]: Does ROOTFS_SOURCE_DIR contain a relative path? And doesn't it work with a relative path?

Comment (LKomaryanskiy) [reply classes/rootfs-image-generator.bbclass:85]: Yes, you're right. Should I add checking the presence SELinux in DISTRO_FEATURES?

Comment (LKomaryanskiy) [reply classes/rootfs-image-generator.bbclass:85]: It contains symlink for rootfs and unfortunately, setfiles doesn't work with symlinks.

Comment (LKomaryanskiy) [reply classes/rootfs-image-generator.bbclass]: I've added checking.

**Review by al1img — APPROVED** [review]

**Pull request merged**